### DISCUSSION TITLE
How to disable multiple categories

### DISCUSSION THREAD

**Discussion by Paultje52** [searxng/searxng#1427] [Q&A]
Hello! I'm new to searxng, I did a clean install using [searxng-docker](https://github.com/searxng/searxng-docker). I want the page to reload when I click a category and "switch" to that category, just like instances running on https://searx.space/. But when I want to select another category, I'm getting this.
![image](https://user-images.githubusercontent.com/31313717/177045982-4351da7a-47d5-479c-bf34-28ed94d0a7a3.png)

Does anyone know how to fix this? `Search on category select` is enabled. Disabling this doesn't change anything.
![image](https://user-images.githubusercontent.com/31313717/177045999-5b479693-c6dd-4e41-a72f-6294a935268a.png)

**Comment by unixfox** [comment]
Do you have javascript enabled?

**Reply by Paultje52** [reply]
Yes, javascript is enabled. 
![image](https://user-images.githubusercontent.com/31313717/177046214-1c90d6b5-61ce-4a77-8254-29e858d31ef0.png)

**Reply by unixfox** [reply]
Something is probably broken when you installed searxng. Can you reproduce this issue on the public instances on https://searx.space?

**Reply by Paultje52** [reply]
I cannot. These are the steps I took for the installation.
- `cd /usr/local && git clone https://github.com/searxng/searxng-docker.git && cd searxng-docker`
- Edit `.env`: uncomment SEARXNG_HOSTNAME and set it to `search.syncbox.nl`
- Generate a key using `sed -i "s|ultrasecretkey|$(openssl rand -hex 32) |g" searxng/settings.yml`
- Set `limiter` to `false` in `searxng/settings.yml` (because it otherwise only gives me `http error 429`)
- Use docker compose: `sudo docker-compose up -d`

I'm routing the traffic using nginx, but that shouldn't change anything, right? Do you maybe know where it went wrong?

**Reply by unixfox** [reply]
check in the devtools of your browser if you see any errors or issues
and check if you can reproduce the issue without using nginx but by directly connecting to the docker container

**Reply by Paultje52** [reply]
I don't see any errors in the dev console. When connecting directly to my ip (still through nginx), I don't experience the issue. Does that mean that the issue is between my server and cloudflare?

**Reply by unixfox** [reply]
probably, I don't know your setup.

**Reply by return42** [reply]
> Does that mean that the issue is between my server and cloudflare?

OT: it's not recommended to use Cloudflare as reverse proxy because it MITM every request, which is a disaster for user's privacy.
